shell/edit/logging.banzaicloud.io.output/index.vue

```diff
@@ -100,17 +100,26 @@ export default {
     }
   },
 
-  created() {
-    this.registerBeforeHook(this.willSave, 'willSave');
-  },
   methods: {
     getComponent(name) {
       return require(`./providers/${ name }`).default;
     },
     launch(provider) {
       this.$refs.tabbed.select(provider.name);
     },
-    willSave() {
+    saveSettings(done) {
+      const t = this.$store.getters['i18n/t'];
+      const urlCheck = ['https://', 'http://'].some(checkValue => this.value.spec['loki'].url.toLowerCase().startsWith(checkValue));
+      const isLokiHttps = this.value.spec['loki'].url ? urlCheck : undefined;
```

Comment:
By having `urlCheck` and `isLokiHttps` as constants, the operations that populate those run every time, therefore breaking the UI when were aren't in a `this.selectedProvider === 'loki'` scenario
![Screenshot 2023-04-21 at 12 53 05](https://user-images.githubusercontent.com/97888974/233629497-95a73bef-1738-4d43-aa99-addbd8874624.png)
![Screenshot 2023-04-21 at 12 53 09](https://user-images.githubusercontent.com/97888974/233629501-270e454b-5a9e-4f34-a42b-631cf8d26fa0.png)